Sources/PointFree/Privacy.swift

```diff
@@ -0,0 +1,174 @@
+import Css
+import CssReset
+import Either
+import Foundation
+import Html
+import HtmlCssSupport
+import HttpPipeline
+import HttpPipelineHtmlSupport
+import Prelude
+import Styleguide
+
+private let title = "Privacy Policy"
+
+let privacyResponse =
+  writeStatus(.ok)
+    >-> map(lower)
+    >>> respond(
+      view: privacyView,
+      layoutData: { currentUser in
+        SimplePageLayoutData(currentUser: currentUser, data: unit, title: title)
+    }
+)
+
+private let privacyView = View<Prelude.Unit> { _ in
+  gridRow([
+    gridColumn(sizes: [.mobile: 12], [
+      div(
```

Comment:
we should treat this page like the account page by giving it `auto` left/right margins to make it a more columnar style page. really we should make a layout to support this or put support into the existing layout. 

this is the grid that does it:

```swift
  gridRow([
    gridColumn(sizes: [.mobile: 12, .desktop: 8], [style(margin(leftRight: .auto))], [
      div([`class`([Class.padding([.mobile: [.all: 3], .desktop: [.all: 4]])])],
```